source/includes/_endpoint-account.md

```diff
@@ -443,3 +443,58 @@ GET /account/teams/
   }
 }
 ```
+
+### Account Collaborators
+
+An account collaborator is a Crunch.io user that is not a member of your account
+and has access to some/any of your account's datasets.
+
+Account admins can visit the account's collaborators catalog to view the list
+of all collaborators for all datasets of the account.
+
+```http
+GET /account/collaborators/
+```
+
+This catalog lists all the users that are not members of the account that have
+access to any of the account's datasets, projects or teams.
+
+Each element in the catalog tuple links to the user's entity endpoint and 
+has the name and email attribute.
+
+```json
+{
+  "element": "shoji:catalog",
+  "index": {
+        "https://app.crunch.io/api/users/cc9161/": {
+          "name": "John doe",
+          "email": "user1@example.com",
+          "active": true,
+        },
+        "https://app.crunch.io/api/users/a598c7/": {
+          "name": "John notdoe",
+          "email": "user2@example.com",
+          "active": true,
+        }
+  }
+}
+```
+
+To disable collaborators, it is possible to PATCH `active` set to `false` for
+users in the catalog.
```

Comment:
Right now it has no effects, it just stores the flag. According to the doc it would mean to remove access from datasets, projects, teams and maybe expropriation as well?